core/domain/interaction_registry.py

```diff
@@ -27,26 +27,34 @@
 from core import utils
 from core.constants import constants
 
+from typing import Dict, List
+
+MYPY = False
+if MYPY: # pragma: no cover
+    from extensions.interactions import base
```

Comment:
Importing it directly into the file results in failing of backend tests. Also, this is imported only for type checking.
```
------------------------------------------------------
    WARNING: FAILED TO RUN core.domain.interaction_registry_test

    This is most likely due to an import error.
------------------------------------------------------
Traceback (most recent call last):
  File "/home/struct_shushi/Desktop/opensource-oppia/oppia/scripts/run_backend_tests.py", line 458, in main
    test_count = int(tests_failed_regex_match.group(1))
AttributeError: 'NoneType' object has no attribute 'group'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/struct_shushi/.pyenv/versions/3.7.10/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/struct_shushi/.pyenv/versions/3.7.10/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/struct_shushi/Desktop/opensource-oppia/oppia/scripts/run_backend_tests.py", line 603, in <module>
    main()
  File "/home/struct_shushi/Desktop/opensource-oppia/oppia/scripts/run_backend_tests.py", line 476, in main
    raise task.exception from e
  File "/home/struct_shushi/Desktop/opensource-oppia/oppia/scripts/concurrent_task_utils.py", line 98, in run
    self.task_results = self.func()
  File "/home/struct_shushi/Desktop/opensource-oppia/oppia/scripts/run_backend_tests.py", line 190, in run
    result = run_shell_cmd(exc_list, env=env)
  File "/home/struct_shushi/Desktop/opensource-oppia/oppia/scripts/run_backend_tests.py", line 160, in run_shell_cmd
    raise Exception('Error %s\n%s' % (p.returncode, result))
Exception: Error 1
Traceback (most recent call last):
  File "/home/struct_shushi/Desktop/opensource-oppia/oppia/core/tests/gae_suite.py", line 125, in <module>
    main()
  File "/home/struct_shushi/Desktop/opensource-oppia/oppia/core/tests/gae_suite.py", line 111, in main
    test_target=parsed_args.test_target,
  File "/home/struct_shushi/Desktop/opensource-oppia/oppia/core/tests/gae_suite.py", line 60, in create_test_suites
    if test_target else
  File "/home/struct_shushi/.pyenv/versions/3.7.10/lib/python3.7/unittest/loader.py", line 154, in loadTestsFromName
    module = __import__(module_name)
  File "/home/struct_shushi/Desktop/opensource-oppia/oppia/core/domain/interaction_registry_test.py", line 27, in <module>
    from core.domain import exp_services
  File "/home/struct_shushi/Desktop/opensource-oppia/oppia/core/domain/exp_services.py", line 41, in <module>
    from core.domain import caching_services
  File "/home/struct_shushi/Desktop/opensource-oppia/oppia/core/domain/caching_services.py", line 25, in <module>
    from core.domain import exp_domain
  File "/home/struct_shushi/Desktop/opensource-oppia/oppia/core/domain/exp_domain.py", line 39, in <module>
    from core.domain import state_domain
  File "/home/struct_shushi/Desktop/opensource-oppia/oppia/core/domain/state_domain.py", line 38, in <module>
    from core.domain import interaction_registry  # pylint: disable=invalid-import-from # isort:skip
  File "/home/struct_shushi/Desktop/opensource-oppia/oppia/core/domain/interaction_registry.py", line 29, in <module>
    from extensions.interactions import base
  File "/home/struct_shushi/Desktop/opensource-oppia/oppia/extensions/interactions/base.py", line 45, in <module>
    from core.domain import visualization_registry
  File "/home/struct_shushi/Desktop/opensource-oppia/oppia/core/domain/visualization_registry.py", line 23, in <module>
    from extensions.visualizations import models
  File "/home/struct_shushi/Desktop/opensource-oppia/oppia/extensions/visualizations/models.py", line 23, in <module>
    from core.domain import calculation_registry
  File "/home/struct_shushi/Desktop/opensource-oppia/oppia/core/domain/calculation_registry.py", line 23, in <module>
    from extensions.answer_summarizers import models
  File "/home/struct_shushi/Desktop/opensource-oppia/oppia/extensions/answer_summarizers/models.py", line 52, in <module>
    exp_domain.EXPLICIT_CLASSIFICATION,
AttributeError: module 'core.domain.exp_domain' has no attribute 'EXPLICIT_CLASSIFICATION'
```